### DISCUSSION TITLE
UTM ON INTEL MAC

### DISCUSSION THREAD

**Discussion by aneeahuja** [utmapp/UTM#4585] [UTM Help (macOS)]
I AM USING UTM WINDOWS ON M1 MAC. BUT I WAS PLANNING TO BUY A INTEL MAC. WILL UTM RUN ON INTEL MAC AND WILL IT RUN X86 VIRTUALIZATION ON INTEL MACS??

**Comment by SamNeve** [comment]
Yes it will, that's what I use now instead of VirtualBox
